test/endecode_test.c

```diff
@@ -89,8 +89,9 @@ static EVP_PKEY *make_key(const char *type, EVP_PKEY *template,
  * (TEST_strn2_eq(), for example).
  */
 
-typedef int (encoder)(void **encoded, long *encoded_len, void *object,
-                      const char *output_type, int selection,
+typedef int (encoder)(void **encoded, long *encoded_len,
```

Comment:
Should this be testing all the extra output structure types?

Reply:
No, I have a separate test for those, in #13262.  The reason I made that a separate PR was that it encompasses both encoders and decoders